docs/reference/enums.md

```diff
@@ -163,14 +163,29 @@
 
 ## PlanUsageType
```

Comment:
old name here, move it to price

Reply:
What do you mean? Should I remove all Plan related enums from the documentation? So far I have aliased PlanUsageType to PriceUsageType and used PriceUsageType where possible

Reply:
I mean you forgot to update the name, and you should make sure to alphabetize it.

Reply:
oh, got it :+1: